lib/cocoapods/installer/xcode/pods_project_generator/target_installer.rb

```diff
@@ -170,6 +170,7 @@ def create_info_plist_file(path, native_target, version, platform, bundle_packag
               add_file_to_support_group(path) if add_to_support_group
 
               native_target.build_configurations.each do |c|
+                next unless c.resolve_build_setting('INFOPLIST_FILE').nil?
```

Comment:
this might not be the right spot to debounce it. This means the file was actually written to disk but not "linked" to the project.

Could we check on this a bit earlier within `pod_target_installer` instead?